specs/capri/README.md

```diff
@@ -0,0 +1 @@
+This folder contains design specs for the Capri release.
```

Comment:
Your PR should not container this change because it is already in master.

Reply:
This is already in the master.  So it should not show up as a new change from you.  You need a rebase.